web-client/src/ducks/findRequests/actions.ts

```diff
@@ -0,0 +1,41 @@
+import { observeFindRequests as observeFindRequestsFunc } from './functions';
```

Comment:
Created new directory for findRequests.  Repeated pattern for myRequests and myOffers.  With new Algolia code, this can be removed.